LuaRules/Utilities/SpecialWeaponCaching.lua

```diff
@@ -15,25 +15,17 @@ for i = 1, #WeaponDefs do
 end
 
 function Spring.Utilities.IsPureSlowWeapon(weaponID)
-	if pureSlowWeapons[weaponID] then
-		return true
-	else
-		return false
-	end
+	return (pureSlowWeapons[weaponID] and true) or false
```

Comment:
how about just
```suggestion
	return pureSlowWeapons[weaponID]
```
(if `false` vs `nil` is a problem then set stuff to false in the table)

Reply:
1aae5a2